CMakeLists.txt

```
@@ -32,6 +32,7 @@ project(openthread-br VERSION 0.2.0)
 
 add_library(otbr-config INTERFACE)
 
+option(OTBR_BACKBONE      "Build Backbone network features" ON)
```

```suggestion
option(OTBR_BACKBONE      "Build Backbone network features" OFF)
```
Better to set the default to `OFF` before we have more tests for the backbone features. Otherwise it may influence the stability of OTBR in produces.